specs/protocol/derivation.md

```diff
@@ -188,6 +189,15 @@ first block of the epoch appears in the very last L1 block of the window. Note t
 applies to _block_ derivation. Sequencer batches can still be derived and tentatively queued
 without deriving blocks from them.
 
+## Protocol Parameters
```

Comment:
I think this section is nice, we will have to remember to update it as more things are added